chapters/statements.tex

```diff
@@ -64,7 +64,7 @@ \subsection{Execution of an algorithm in a model}\label{execution-of-an-algorith
 This is usually the case, except for algorithms with \lstinline!when!-statements, and especially for initial algorithms.
 \end{nonnormative}
 
-\subsection{Execution of the algorithm in a function}\label{execution-of-the-algorithm-in-a-function}
+\subsection{The algorithm in a function}\label{execution-of-the-algorithm-in-a-function}\label{the-algorithm-in-a-function}
```

Comment:
Same question as above: are algorithm and function are used as keywords?

Reply:
Will also change.

Reply:
Resolving